src/renderer/i18n/en.js

```diff
@@ -165,7 +165,8 @@ const English = {
     connect: "Connect",
     disconnect: "Disconnect",
     scan: "Scan keyboards",
-    permissionError: `Insufficient permissions, please make sure the device file is read- and writeable!`
+    installUdevRules: "Install Udev rules",
```

Comment:
title "Fix permissions error"

Reply:
"Grant permissions"?

Reply:
In general, I try to go for simple, friendly language - and "fix" matches with the explanation just above the message.

Reply:
Ah, I didn't notice your other comment about the error message itself! In that context, "Fix permissions error" works just fine. Sorry!